README.md

```diff
@@ -2,75 +2,35 @@
 
 ## Installation
 
-### macOS
-
-1. Download `Hyper-3.1.1-serenade.dmg` from https://github.com/serenadeai/serenade-hyper/releases and move `Hyper.app` to your Applications folder.
-    - This version is necessary since it includes an updated version of xterm.js.
-    - If you already have a version of Hyper installed, you'll need to update your configuration to include:
-      ```
-      plugins: [
-         ...
-        'serenade-hyper'
-      ],
-      ```
-1. Launch Hyper, and use the menu item Plugins > Update to automatically download the Serenade plugin.
-1. From your shell, run `~/.hyper_plugins/node_modules/serenade-hyper/install.sh` to automatically add the shell integration to your shell profile.
-    - Alternatively, you can run `cat ~/.hyper_plugins/node_modules/serenade-hyper/install.sh` to see the commands in the script and run them manually.
-1. Restart Hyper to load the plugin and shell integration.
-
-### Windows
-
-1. Download and run `Hyper Setup 3.1.1-serenade.exe` from https://github.com/serenadeai/serenade-hyper/releases.
-    - This version is necessary since it includes an updated version of xterm.js.
-1. Launch Hyper, and use the menu item Plugins > Update to automatically download the Serenade plugin.
-1. You will likely need to change Hyper's configuration file to point to your shell with Edit > Preferences. The default configuration file will have examples in the comments above the `shell` entry. For example, Git Bash will require the following:
-    ```
-    shell: 'C:\\Program Files\\Git\\bin\\bash.exe',
```

Comment:
This might still be a helpful step for people on Windows since it's not immediately obvious what to do when Hyper opens to the default command prompt?